### ISSUE TITLE
loss计算问题

### ISSUE THREAD

**Issue by laketitikakaou** [lambdaji/tf_repos#20]
loss = tf.reduce_mean(tf.nn.sigmoid_cross_entropy_with_logits(logits=y, labels=labels)) 
这里的计算loss的时候，预测值使用的是y,在代码里面y是未经过sigmoid激活函数的一个输出，想请教一下这样做的原因，因为通常来说分类任务都是会经过sigmoid做预测的。
PS：比较困惑的是，做实验发现不经过sigmoid网络优化的更好，但是一经过sigmoid之后就会差很多。
谢谢！
